final/implementation/dynsem-eval-strat.tex

```diff
@@ -0,0 +1,136 @@
+\section{Dynsem Evaluation Strategy}
+\label{sec:dynsem-eval-strat}
+An implementation for the \texttt{IEvaluationStrategy} as discussed in
+\cref{sec:eval-strat} has been made for languages that specify their dynamic
+semantics using DynSem. The implementation class, called
+\texttt{``DynSemEvaluationStrategy''} (see \cref{fig:uml-dynsem-eval-strat}),
+evaluates REPL-specific DynSem rules and maintains the environment in which
+evaluation takes place.
+
+This section first goes over the configuration interface for DynSem that is
+provided by the REPL. After that, the implementation of DynSem and of the
+evaluation strategy are discussed in more detail.
+
+\subsection{The REPL configuration interface}
+\label{sec:impl-repl-spec}
+To evaluate a program in their language within the REPL, the language designer
+has to make two sorts of configurations for the REPL in their DynSem
```

Comment:
configurations (plural) sounds strange to me, but it is correct, so up to you what you want to do with this :P